src/layer/audio-source.ts

```diff
@@ -85,9 +85,12 @@ function AudioSourceMixin<OptionsSuperclass extends BaseOptions> (superclass: Co
       applyOptions(options, this)
 
       const load = () => {
-        // TODO:              && ?
-        if ((options.duration || (this.source.duration - this.sourceStartTime)) < 0) {
-          throw new Error('Invalid options.duration or options.sourceStartTime')
+        if (options.duration < 0) {
+          throw new Error('Invalid options.duration. It must be a non-negative value.')
+        }
+
+        if (this.source.duration < this.sourceStartTime) {
+          throw new Error('Invalid options.sourceStartTime. It must greater than options.source.duration.')
```

Comment:
This should be the opposite:

> options.sourceStartTime cannot exceed options.source.duration.

Reply:
I think it's alright. If sourceStartTime is greater than source.duration, it will throw an error, which implies it won't throw an error if sourceStartTime is less than or equal to duration

Reply:
Hi, sorry for the late reply, starting a new job. Yea sourceStartTime must be less than or equal to duration, but the error message says it must be greater than duration.

Reply:
Congrats on the new job! Alright, I will fix it up immediately

Reply:
Thanks!!

I meant you should change the error message to be the opposite of what it was. The condition itself was correct. So,

```ts
if (this.sourceStartTime > this.source.duration) {
  throw new Error('options.sourceStartTime cannot exceed options.source.duration')
}
```

Reply:
Hi! Sorry, I have been sick terribly so couldn't attend this issue earlier.

Ah, you meant the Error message itself. I feel dumb now not gonna lie. Let me fix that up real quick, and apologies for the carelessness